src/types/ContributorTypes.ts

```diff
@@ -13,6 +13,14 @@ export interface ContributorType {
   url?: string;
   affiliations?: Affiliation[];
   require_higher_authorization?: boolean;
+  badge_type?: BadgeType;
+}
+
+export enum BadgeType {
```

Comment:
kunne nesten ha kalt typen for ContributorStatus elns ? den brukes ikke bare til badge-visning men også for styling av headeren